test_linux.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+timeout 20s ./zubax-kucher
+exit_status=$?
+if [ $exit_status -eq 124 ]; then
```

Comment:
On GNU-based systems, any error code except zero indicates an error, so the test against 124 is not only unnecessary, it is actually dangerous because it is a case of fragile logic: the tool may return anything other than 124 to indicate an error, but this script would fail to catch that because it is hard coded to expect exactly 124.

Consider this approach instead: https://github.com/UAVCAN/pyuavcan/blob/d348a81fbfb96350305e18bf2188deecadde01d6/test.sh#L26-L49

Reply:
My goal was to run the program for 20 seconds and if no error occured in kucher, it would return exit code 0. timeout return exit code 124 when the timer is over so it would make the build fail.